sktime/transformations/base.py

```diff
@@ -48,93 +104,567 @@ class BaseTransformer(BaseEstimator):
         "skip-inverse-transform": False,  # is inverse-transform skipped when called?
     }
 
+    # allowed mtypes for transformers - Series and Panel
+    ALLOWED_INPUT_MTYPES = [
+        "pd.Series",
+        "pd.DataFrame",
+        "np.ndarray",
+        "nested_univ",
+        "numpy3D",
+        # "numpyflat",
+        "pd-multiindex",
+        # "pd-wide",
+        # "pd-long",
+        "df-list",
+    ]
+
     def __init__(self):
+        self._is_fitted = False
```

Comment:
Not needed set in BaseEstimator init

Reply:
yes, that's actually right! Thanks for pointing out.

Reply:
Fixed